sencha-workspace/SlateAdmin/app/controller/progress/interims/Report.js

```diff
@@ -220,12 +230,39 @@ Ext.define('SlateAdmin.controller.progress.interims.Report', {
         studentsGrid.enable();
 
         // configure and load stores
-        studentsStore.getProxy().setUrl('/sections/'+section.get('Code')+'/students');
+        studentsStore.getProxy().setUrl('/sections/'+sectionCode+'/students');
         studentsStore.load();
 
         reportsProxy.setExtraParam('term', termHandle);
         reportsProxy.setExtraParam('course_section', sectionCode);
         reportsStore.load();
+
+        sectionNotesForm.enable();
+        sectionNotesForm.setLoading('Loading notes&hellip;');
+
+        SectionNotesModel.getProxy().createOperation('read', {
+            params: {
+                term: termHandle,
+                course_section: sectionCode
+            },
+            callback: function(sectionNotes, operation, success) {
+                if (!success) {
+                    return;
+                }
+
+                if (!(sectionNotes = sectionNotes[0])) {
+                    sectionNotes = new SectionNotesModel({
+                        TermID: term.getId(),
+                        CourseSectionID: section.getId()
```

Comment:
@nbey creating new section notes does not work currently...below on line 351 we're getting `sectionDataRecord.get('SectionID')` and aborting if it doesn't resolve to a section, but here we're setting `sectionNotes.get('CourseSectionID')`